plugins/catalog-backend/src/ingestion/processors/github/github.ts

```diff
@@ -178,12 +184,18 @@ export async function getOrganizationTeams(
   // Gets populated inside the mapper below
   const groupMemberUsers = new Map<string, string[]>();
 
+  let orgName: string | undefined;
+  let orgDescription: string | undefined;
+
+  const orgSlug = orgNamespace || org;
```

Comment:
I wonder if we should perhaps just stick to only using the `orgNamespace`, and letting people set it in config explicitly if desired? We would definitely want to avoid having existing org data in peoples catalogs be shuffled around